src/schema/validator.js

```diff
@@ -5,7 +5,7 @@ import { getDefaultConfigValue } from 'yargs-options';
 import { deepPatch } from 'schema/deepmerge';
 import schemaObject from 'schema/imported/manifest';
 import themeSchemaObject from 'schema/imported/theme';
-import messagesSchemaObject from 'schema/messages';
+import messagesSchemaObject from 'schema/i18nMessages';
```

Comment:
The usual convention for the file name is to use underscore instead of camelCase, e.g. `i18n_messages.json`.

Reply:
Done.